en/22/10.md

```diff
@@ -0,0 +1,291 @@
+              let verified_target_dna = target_dna % max_dna_value;
+              let new_dna = (my_zombie.dna + verified_target_dna) / 2;
+              self.create_zombie(caller, ManagedBuffer::from(b"NoName"), new_dna);
+          }
+
+          #[proxy]
+          fn kitty_proxy(&self, to: ManagedAddress) -> crypto_kitties_proxy::Proxy<Self::Api>;
+      }
+---
+
+It's time to feed our zombies! And what do zombies like to eat most?
+
+Well it just so happens that CryptoZombies love to eat...
+
+**CryptoKitties!** 😱😱😱
+
+(Yes, I'm serious 😆 )
+
+In order to do this we'll need to read the kittyDna from the CryptoKitties smart contract. We can do that because the CryptoKitties data is stored openly on the blockchain. Isn't the blockchain cool?!
+
+Don't worry — our game isn't actually going to hurt anyone's CryptoKitty. We're only *reading* the CryptoKitties data, we're not able to actually delete it 😉
+
+## Interacting with other contracts
+
+For our contract to talk to another contract on the blockchain that we don't own, first we need to define a **_Proxy_**.
+
+Let's look at a simple example. Say there was a contract on the blockchain that looked like this:
+
+```
+#[multiversx_sc::contract]
+pub trait Adder {
+   
+    ...
+
+    #[endpoint]
+    fn add(&self, a: BigUint, b: BigUint) -> BigUint {
+        a + b
+    }
+}
+```
+
+This would be a simple contract where you would sum up 2 numbers and returns their sum.
+
+Now let's say we had an external contract that wanted to read the data in this contract using the `add` function. 
+
+First we'd have to define an **_Proxy_** of the `Adder` contract:
+
+```
+mod my_adder_proxy {
+    multiversx_sc::imports!();
+
+    #[multiversx_sc::proxy]
+    pub trait Adder {        
+      #[endpoint]
+      fn add(&self, a: BigUint, b: BigUint) -> BigUint
+    }
+}
+
+#[multiversx_sc::contract]
+pub trait MyContract {
+    
+    ...
+
+    #[proxy]
+    fn adder_proxy(&self, to: ManagedAddress) -> my_adder_proxy::Proxy<Self::Api>;
+}
+```
+Notice that this looks like defining a contract except the `#[multiversx_sc::proxy]` annotation of the trait and a few other differences. For one, we're only declaring the functions we want to interact with — in this case `add` — and we don't mention any of the other functions or state variables.
+
+Secondly, we're not defining the function bodies. Instead of curly braces (`{` and `}`), we're simply ending the function declaration with a semi-colon (`;`).
+
+So it kind of looks like a contract skeleton. This is how the compiler knows it's an interface.
+
+By including this interface in our dapp's code our contract knows what the other contract's functions look like, how to call them, and what sort of response to expect.
+
+Next we might see inside our contract / module another function with the `#[proxy]` annotation. It looks kind of similar to how we define a storage mapper, except the return type being of type `Proxy` from our proxy module defined before.
```

Comment:
This seems out of place here. Please consider telling the name of this other function you're referring to and why is this relevant.